ibm/mas_devops/roles/gencfg_mongo/defaults/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+#mongo configuration
+# -----------------------------------------------------------------------------
+# Currently supported providers are documentdb,mongo
+mongo_namespace: "{{ lookup('env', 'MONGODB_NAMESPACE') | default('mongoce', true) }}"
```

Comment:
Can we keep consistency cross the prefixes?
in `mongodb` role we can all env vars starting with `MONGODB` can we have same standard in this new role?

Reply:
@andrercm - I made the suggested changes